gammapy/modeling/models/core.py

```diff
@@ -1164,19 +1164,29 @@ def positions(self):
 
         return SkyCoord(positions)
 
-    def to_regions(self):
+    def to_regions(self, x_width=None, **kwargs):
         """Return a list of the regions for the spatial models.
 
+        Parameters
+        ----------
+        x_width : float, optional
+            Number of :math:`sigma` for `GaussianSpatialModel`
```

Comment:
```suggestion
            Number of ``sigma`` for `GaussianSpatialModel`
```

I would do it this way as it is the name of the variable